app/templates/main/map.html

```diff
@@ -0,0 +1,106 @@
+            year = parseInt((dateString.split(' '))[0].split('-')[0]);
+            month = parseInt((dateString.split(' '))[0].split('-')[1])-1;
+            day = parseInt((dateString.split(' '))[0].split('-')[2]);
+            incidentDate = new Date(year, month, day);
+            if (incidentDate.getTime() < minDate.getTime())
+                minDate.setTime(incidentDate.getTime());
+            //marker.setMap(map);
+            var contentString = '<div id="content">' +
+                                '<div id="siteNotice">' +
+                                '</div>' +
+                                '<h1 id="firstHeading" class="firstHeading">Vehicle ID: ' + '{{ report.vehicle_id }}' +'</h1>' +
+                                '<div id="bodyContent">' +
+                                '<p>License Plate: ' + '{{ report.license_plate }}' + '</p>' +
+                                '<p>Date: ' + '{{ report.date }}' + '</p>' +
+                                '<p>Duration: ' + '{{ report.duration }}' + '</p>' +
+                                '<p>Agency: ' + '{{ report.agency.name }}' + '</p>' +
+                                '<p>Link to Picture: ' + '{{ report.picture_url }}' + '</p>' +
+                                '<p>Description: ' + '{{ report.description }}' + '</p>' +
+                                '</div>' +
+                                '</div>';
+            bounds.extend(marker.getPosition());
+            markerWrapper = new MarkerWrapper(marker, incidentDate, contentString);
+            markerWrappers.push(markerWrapper);
+            var infoWindow = new google.maps.InfoWindow({
+                content: contentString
+            });
+
+            // Add click listener to marker for displaying infoWindow
+            marker.addListener('click', function() {
+                infoWindow.open(map, marker);
+            });
+        });
+        {% endfor %}
+        var mapProp = {
+                mapTypeId: google.maps.MapTypeId.ROADMAP,
+                mapTypeControl: false,
+                streetViewControl: false
+        };
+        map=new google.maps.Map($('#googleMap')[0],mapProp);
+
+        // Use HTML geolocation to center map if possible
+        if (navigator.geolocation) {
+            navigator.geolocation.getCurrentPosition(function(position) {
+                var pos_center = {
+                    lat: position.coords.latitude,
+                    lng: position.coords.longitude
+                };
+                map.setCenter(pos_center);
+            });
+        }
+        storeMarkerState(markerWrappers, map, minDate, bounds);
+        initializeDateSlider();
+        console.log("Done\n");
+    }
+    // Add the map to the window
+    google.maps.event.addDomListener(window, 'load', initializeMapAndMarkers);
+
+    // Set the height of the googleMap to the height of the window (less
+    // any elements above the googleMap)
+    $(function () {
```

Comment:
As I commented above, putting all of the code in this file in a $(document).ready causes scope errors